### DISCUSSION TITLE
No VSCODE type checking support

### DISCUSSION THREAD

**Discussion by gp-slick-coder** [sdispater/pendulum#673] [General]
<img width="566" alt="image" src="https://user-images.githubusercontent.com/1906169/201538183-59a880bb-b2a2-4fe0-9d8c-6e55f3903b51.png">

<img width="504" alt="image" src="https://user-images.githubusercontent.com/1906169/201538275-08e022f8-7e10-4de5-895e-458afb3f79c4.png">

**Comment by Secrus** [comment]
What version of Pendulum are you on?

**Reply by gp-slick-coder** [reply]
pip show pendulum
      Name: pendulum
      Version: 2.1.2
      Summary: Python datetimes made easy
      Home-page: https://pendulum.eustace.io
      Author: Sébastien Eustace
      Author-email: sebastien@eustace.io
      License: MIT
      Location: /home/george/repos/source-backend/.venv/lib/python3.8/site-packages
      Requires: python-dateutil, pytzdata
      Required-by:

**Reply by Secrus** [reply]
Versions 2.x do not provide typing information, so Pylance is correct here.
